.circleci/config.yml

```diff
@@ -139,13 +139,35 @@ jobs:
           paths:
             - "*"
 
+  build_x86_archlinux:
+    docker:
+      - image: archlinux/base
+    environment:
+      TERM: xterm
+    steps:
+      - run:
+          name: Install build dependencies
+          command: |
+            pacman --noconfirm -Syu --noprogressbar --needed base-devel boost cmake z3 cvc4 git openssh tar
+      - checkout
+      - run: *setup_prerelease_commit_hash
+      - run: *run_build
+      - store_artifacts: *solc_artifact
+      - persist_to_workspace:
+          root: build
+          paths:
+            - solc/solc
+            - test/soltest
+            - test/tools/solfuzzer
+
   build_x86_clang7:
     docker:
       - image: buildpack-deps:cosmic
     environment:
       TERM: xterm
       CC: /usr/bin/clang-7
       CXX: /usr/bin/clang++-7
+      IPC_ENABLED: false
```

Comment:
Why is this variable set for the build?

Reply:
Ah - because I wanted to disable IPC builds and still use the ``scripts/tests.sh`` script, but then noticed the script doesn't support setting that variable externally - I just forgot to remove it here afterwards - one second...

Reply:
Hm... that's weird - it's not even the right build...

Reply:
Anyways - gone now.